lib/rules/max-statements-per-line.js

```diff
@@ -35,11 +35,12 @@ module.exports = {
         const sourceCode = context.getSourceCode(),
             options = context.options[0] || {},
             maxStatementsPerLine = typeof options.max !== "undefined" ? options.max : 1,
-            message = "This line has too many statements. Maximum allowed is {{maxStatementsPerLine}}.",
-            data = { maxStatementsPerLine };
+            emptyBlockMessage = "This line has an empty block, but no statements are allowed.",
```

Comment:
I think we should not add a special message and rather remove the empty block check later (see https://github.com/eslint/eslint/issues/7051). If you can revert this piece, I think the rest is good to go.